###
### THIS FILE IS AUTOMATICALLY GENERATED by ./spur_controller_configuration_gen.sh /home/catkin_ws/ws_tork/src/spur/spur_controller/config Sat Jan 24 23:57:51 JST 2015
###

joint_state_publisher:
   controller:
       package: dynamixel_controllers
       module: joint_state_publisher
       type: JointStatePublisher


# fl_rotation_joint
fl_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: fl_rotation_joint
    joint_speed: 2.0
    motor:
        id: 1
        init: 2048
        min: 0
        max: 4095

# bl_rotation_joint
bl_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: bl_rotation_joint
    joint_speed: 2.0
    motor:
        id: 2
        init: 2048
        min: 0
        max: 4095

# br_rotation_joint
br_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: br_rotation_joint
    joint_speed: 2.0
    motor:
        id: 3
        init: 2048
        min: 0
        max: 4095

# fr_rotation_joint
fr_rotation_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: fr_rotation_joint
    joint_speed: 2.0
    motor:
        id: 4
        init: 2048
        min: 0
        max: 4095

# fl_wheel_joint
fl_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fl_wheel_joint
    joint_speed: 2.0
    motor:
        id: 5
        init: 0 
        min: -1024 
        max: 1024

# bl_wheel_joint
bl_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: bl_wheel_joint
    joint_speed: 2.0
    motor:
        id: 6
        init: 0 
        min: -1024 
        max: 1024

# br_wheel_joint
br_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: br_wheel_joint
    joint_speed: 2.0
    motor:
        id: 7
        init: 0 
        min: -1024 
        max: 1024

# fr_wheel_joint
fr_wheel_joint_velocity_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: fr_wheel_joint
    joint_speed: 2.0
    motor:
        id: 8
        init: 0 
        min: -1024 
        max: 1024

# larm_elbow_p_joint
larm_elbow_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_elbow_p_joint
    joint_speed: 2.0
    motor:
        id: 9
        init: 2048
        min: 0
        max: 4095

# larm_shoulder_p_joint
larm_shoulder_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_shoulder_p_joint
    joint_speed: 2.0
    motor:
        id: 10
        init: 2048
        min: 0
        max: 4095

# larm_shoulder_r_joint
larm_shoulder_r_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_shoulder_r_joint
    joint_speed: 2.0
    motor:
        id: 11
        init: 2048
        min: 0
        max: 4095

# larm_shoulder_y_joint
larm_shoulder_y_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_shoulder_y_joint
    joint_speed: 2.0
    motor:
        id: 12
        init: 2048
        min: 0
        max: 4095

# larm_wrist_p_joint
larm_wrist_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_wrist_p_joint
    joint_speed: 2.0
    motor:
        id: 13
        init: 2048
        min: 0
        max: 4095

# larm_wrist_r_joint
larm_wrist_r_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: larm_wrist_r_joint
    joint_speed: 2.0
    motor:
        id: 14
        init: 2048
        min: 0
        max: 4095

# rarm_elbow_p_joint
rarm_elbow_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_elbow_p_joint
    joint_speed: 2.0
    motor:
        id: 15
        init: 2048
        min: 0
        max: 4095

# rarm_shoulder_p_joint
rarm_shoulder_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_shoulder_p_joint
    joint_speed: 2.0
    motor:
        id: 16
        init: 2048
        min: 0
        max: 4095

# rarm_shoulder_r_joint
rarm_shoulder_r_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_shoulder_r_joint
    joint_speed: 2.0
    motor:
        id: 17
        init: 2048
        min: 0
        max: 4095

# rarm_shoulder_y_joint
rarm_shoulder_y_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_shoulder_y_joint
    joint_speed: 2.0
    motor:
        id: 18
        init: 2048
        min: 0
        max: 4095

# rarm_wrist_p_joint
rarm_wrist_p_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_wrist_p_joint
    joint_speed: 2.0
    motor:
        id: 19
        init: 2048
        min: 0
        max: 4095

# rarm_wrist_r_joint
rarm_wrist_r_joint_position_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: rarm_wrist_r_joint
    joint_speed: 2.0
    motor:
        id: 20
        init: 2048
        min: 0
        max: 4095
